### PR TITLE
N36: Renet

### PR DESCRIPTION
**Pull request by lucaspoffo**
Part of https://github.com/rust-gamedev/rust-gamedev.github.io/issues/1098

This is my first contribution to the newsletter, so feedbacks are appreciated :)

### CODE REVIEW

File: content/news/036/index.md [hunk 1]
```diff
@@ -107,6 +107,33 @@ post on the Discussion boards!
...
+- Connection management
+- Authentication and encrypted connections
+- Communication through multiple types of channels:
+  - Reliable Ordered: guarantee ordering and delivery of all messages
+  - Unreliable Unordered: no guarantee of delivery or ordering of messages
+  - Block Reliable: for bigger messages, such as level initialization
+- Packet fragmentation and reassembly
+
+Renet comes with [bevy_renet] a plugin for the Bevy engine and also with
+[renet_visualizer] an egui interface to visualize network metrics.
```
Comment (17cupsofcoffee) [inline content/news/036/index.md:130]:
```suggestion
Renet comes with [bevy_renet], a plugin for the Bevy engine, and also with
[renet_visualizer], an egui interface to visualize network metrics.
```

**Review by 17cupsofcoffee — APPROVED** [review]
:+1: LGTM!

cc @ozkriff: We usually prefer to avoid nested lists, but not sure if there's a nicer way of formatting this one?

**Review by ozkriff — APPROVED** [review]
yeah, this specific nested list looks harmless

**Comment by ozkriff** [comment]
Thanks for the PR!

**Pull request merged**